pyodide-build/pyodide_build/_f2c_fixes.py

```diff
@@ -226,6 +229,45 @@ def fix_f2c_output(f2c_output_path: str):
             for line in lines
         ]
 
+    # Substitute back the dummy fixed array sizes. We also have to remove the
+    # "static" storage specifier since variable sized arrays can't have static
+    # storage.
```

Comment:
I can't wait for lfortran to have a good compiler with WASM support !

Thanks for finding this workaround!